Dungeon_of_Munchkinomicon.py

```diff
@@ -20,19 +20,28 @@ def death(name, reason):
 	print "You died because %s. What did you think would happen?" % reason
 	redo(name)
 	
+def getConfirmation(ask):
+        print ask
+        choice = raw_input(prompt).lower()

```

Comment:
.lower() is a function that can be called from strings that lower-cases the given string. This resolves the need to check for both "yes" and 'yes'. Check out the lower function [here.](https://docs.python.org/2/library/string.html#string.lower)